src/Utils/Rotation.cpp

```diff
@@ -455,89 +454,85 @@ void utils::Rotation::checkUnitary()
             utils::Matrix3d normalEq = Mm.transpose() * Mm;
             utils::Vector3d eigen_vals;
             utils::Matrix3d eigen_vecs;
-
-            /////////////////////
-            //eigen_sym(normalEq, eigen_vals, eigen_vecs);
-
             utils::Matrix3d V;
             utils::Vector3d d;
 
             int n = 3;
 
             utils::Vector3d e;
-            for (int i = 0; i < n; i++) {
-                for (int j = 0; j < n; j++) {
+            for (size_t i = 0; i < n; ++i) {
+                for (size_t j = 0; j < n; ++j) {
                     V(i, j) = normalEq(i, j);
                 }
             }
```

Comment:
This feels the same as `V = normalEq;`?

Reply:
I was afraid of 'pointer effect' XD Sorry you have to babysit my c++!